converters/fromZigbee.js

```diff
@@ -4759,6 +4759,23 @@ const converters = {
             return {humidity: calibrateAndPrecisionRoundOptions(humidity, options, 'humidity')};
         },
     },
+    3400_command_arm: {
+        cluster: 'ssIasAce',
+        type: 'commandArm',
+        convert: (model, msg, publish, options, meta) => {
+            const lookup = {
+                0: 'disarm',
+                1: 'arm_day_zones',
+                2: 'arm_night_zones',
+                3: 'arm_all_zones',
+            };
+            const keycode = String.fromCharCode(msg.data['u1'],msg.data['u2'],msg.data['u3'],msg.data['u4'],msg.data['u5'],msg.data['u6']);
```

Comment:
Did you make any change to zigbee-herdsman? Because I don't expect that you get `"unknown1":6,"u1":49,"u2":49,"u3":49,"u4":49,"u5":49,"u6":49` (only `armmode`)

Reply:
I edited `ssIasAce` in `node_modules/zigbee-herdsman/dist/zcl/definition/cluster.js`

to:

```
ssIasAce: {
        ID: 1281,
        attributes: {},
        commands: {
            arm: {
                ID: 0,
                parameters: [
                    { name: 'armmode', type: dataType_1.default.uint8 },
                    { name: 'unknown1', type: dataType_1.default.uint8 },
                    { name: 'u1', type: dataType_1.default.uint8 },
                    { name: 'u2', type: dataType_1.default.uint8 },
                    { name: 'u3', type: dataType_1.default.uint8 },
                    { name: 'u4', type: dataType_1.default.uint8 },
                    { name: 'u5', type: dataType_1.default.uint8 },
                    { name: 'u6', type: dataType_1.default.uint8 },
                ],
            },
```

Now i reverted back to original cluster.js. Here is the log:

```
zigbee2mqtt:debug 2020-06-11 20:07:46: Received Zigbee message from '0x000d6f000d8a5f63', type 'commandArm', cluster 'ssIasAce', data '{"armmode":0}' from endpoint 1 with groupID 0
zigbee2mqtt:info  2020-06-11 20:07:46: MQTT publish: topic 'zigbee2mqtt/0x000d6f000d8a5f63', payload '{"linkquality":7,"temperature":25.5,"keycode":"\u0000\u0000\u0000\u0000\u0000\u0000","voltage":2900,"action":"disarm"}'
zigbee2mqtt:info  2020-06-11 20:07:46: MQTT publish: topic 'zigbee2mqtt/0x000d6f000d8a5f63', payload '{"linkquality":7,"temperature":25.5,"keycode":"\u0000\u0000\u0000\u0000\u0000\u0000","voltage":2900,"action":""}'
zigbee2mqtt:info  2020-06-11 20:07:46: MQTT publish: topic 'zigbee2mqtt/0x000d6f000d8a5f63/action', payload 'disarm'
zigbee2mqtt:debug 2020-06-11 20:07:58: Device '0x000d6f000d8a5f63' announced itself
zigbee2mqtt:info  2020-06-11 20:07:58: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"type":"device_announced","message":"announce","meta":{"friendly_name":"0x000d6f000d8a5f63"}}'
```

and with `DEBUG=zigbee-herdsman* npm start`:

```
zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,23,68,129,0,0,1,5,255,61,1,1,0,34,0,251,41,79,0,0,3,1,98,7,255,61,29,19] +5s
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,23,68,129,0,0,1,5,255,61,1,1,0,34,0,251,41,79,0,0,3,1,98,7,255,61,29,19] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 23 - 2 - 4 - 129 - [0,0,1,5,255,61,1,1,0,34,0,251,41,79,0,0,3,1,98,7,255,61,29] - 19 +0ms
  zigbee-herdsman:adapter:zStack:znp:AREQ <-- AF - incomingMsg - {"groupid":0,"clusterid":1281,"srcaddr":15871,"srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"linkquality":34,"securityuse":0,"timestamp":5188091,"transseqnumber":0,"len":3,"data":{"type":"Buffer","data":[1,98,7]}} +5s
  zigbee-herdsman:controller:log Received 'zcl' data '{"frame":{"Header":{"frameControl":{"frameType":1,"manufacturerSpecific":false,"direction":0,"disableDefaultResponse":false,"reservedBits":0},"transactionSequenceNumber":98,"manufacturerCode":null,"commandIdentifier":7},"Payload":{}},"address":15871,"endpoint":1,"linkquality":34,"groupID":0}' +5s
  zigbee-herdsman:controller:log Skipping command 'getPanelStatus' because it is missing from the lookup +0ms
  zigbee-herdsman:controller:endpoint DefaultResponse 0x000d6f000d8a5f63/1 1281(7, {"timeout":10000,"disableResponse":false,"disableDefaultResponse":true,"direction":1,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null}) +0ms
  zigbee-herdsman:adapter:zStack:znp:SREQ --> AF - dataRequest - {"dstaddr":15871,"destendpoint":1,"srcendpoint":1,"clusterid":1281,"transid":2,"options":0,"radius":30,"len":5,"data":{"type":"Buffer","data":[24,98,11,7,0]}} +5s
  zigbee-herdsman:adapter:zStack:unpi:writer --> frame [254,15,36,1,255,61,1,1,1,5,2,0,30,5,24,98,11,7,0,131] +5s
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +5ms
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,23,68,129,0,0,1,5,255,61,1,1,0,34,0,8,42,79,0,0,3,1,99,7,255,61,29,226] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,23,68,129,0,0,1,5,255,61,1,1,0,34,0,8,42,79,0,0,3,1,99,7,255,61,29,226] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 23 - 2 - 4 - 129 - [0,0,1,5,255,61,1,1,0,34,0,8,42,79,0,0,3,1,99,7,255,61,29] - 226 +0ms
  zigbee-herdsman:adapter:zStack:znp:AREQ <-- AF - incomingMsg - {"groupid":0,"clusterid":1281,"srcaddr":15871,"srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"linkquality":34,"securityuse":0,"timestamp":5188104,"transseqnumber":0,"len":3,"data":{"type":"Buffer","data":[1,99,7]}} +4ms
  zigbee-herdsman:controller:log Received 'zcl' data '{"frame":{"Header":{"frameControl":{"frameType":1,"manufacturerSpecific":false,"direction":0,"disableDefaultResponse":false,"reservedBits":0},"transactionSequenceNumber":99,"manufacturerCode":null,"commandIdentifier":7},"Payload":{}},"address":15871,"endpoint":1,"linkquality":34,"groupID":0}' +4ms
  zigbee-herdsman:controller:log Skipping command 'getPanelStatus' because it is missing from the lookup +0ms
  zigbee-herdsman:controller:endpoint DefaultResponse 0x000d6f000d8a5f63/1 1281(7, {"timeout":10000,"disableResponse":false,"disableDefaultResponse":true,"direction":1,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null}) +3ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +1ms
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,1,100,1,0,100] +7ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,1,100,1,0,100] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 1 - 3 - 4 - 1 - [0] - 100 +0ms
  zigbee-herdsman:adapter:zStack:znp:SRSP <-- AF - dataRequest - {"status":0} +5s
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +1ms
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,3,68,128,0,1,2,196] +2s
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,3,68,128,0,1,2,196] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 3 - 2 - 4 - 128 - [0,1,2] - 196 +0ms
  zigbee-herdsman:adapter:zStack:znp:AREQ <-- AF - dataConfirm - {"status":0,"endpoint":1,"transid":2} +2s
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +0ms
  zigbee-herdsman:adapter:zStack:znp:SREQ --> AF - dataRequest - {"dstaddr":15871,"destendpoint":1,"srcendpoint":1,"clusterid":1281,"transid":3,"options":0,"radius":30,"len":5,"data":{"type":"Buffer","data":[24,99,11,7,0]}} +2s
  zigbee-herdsman:adapter:zStack:unpi:writer --> frame [254,15,36,1,255,61,1,1,1,5,3,0,30,5,24,99,11,7,0,131] +2s
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,1,100,1,0,100] +8ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,1,100,1,0,100] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 1 - 3 - 4 - 1 - [0] - 100 +0ms
  zigbee-herdsman:adapter:zStack:znp:SRSP <-- AF - dataRequest - {"status":0} +2s
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +1ms
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,32,68,129,0,0,1,5,255,61,1,1,0,5,0,27,84,79,0,0,12,1,100,0,0,6,49,50,51,52,53,54,0,255,61,29,145] +1s
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,32,68,129,0,0,1,5,255,61,1,1,0,5,0,27,84,79,0,0,12,1,100,0,0,6,49,50,51,52,53,54,0,255,61,29,145] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 32 - 2 - 4 - 129 - [0,0,1,5,255,61,1,1,0,5,0,27,84,79,0,0,12,1,100,0,0,6,49,50,51,52,53,54,0,255,61,29] - 145 +0ms
  zigbee-herdsman:adapter:zStack:znp:AREQ <-- AF - incomingMsg - {"groupid":0,"clusterid":1281,"srcaddr":15871,"srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"linkquality":5,"securityuse":0,"timestamp":5198875,"transseqnumber":0,"len":12,"data":{"type":"Buffer","data":[1,100,0,0,6,49,50,51,52,53,54,0]}} +1s
  zigbee-herdsman:controller:log Received 'zcl' data '{"frame":{"Header":{"frameControl":{"frameType":1,"manufacturerSpecific":false,"direction":0,"disableDefaultResponse":false,"reservedBits":0},"transactionSequenceNumber":100,"manufacturerCode":null,"commandIdentifier":0},"Payload":{"armmode":0}},"address":15871,"endpoint":1,"linkquality":5,"groupID":0}' +3s
zigbee2mqtt:debug 2020-06-11 20:10:34: Received Zigbee message from '0x000d6f000d8a5f63', type 'commandArm', cluster 'ssIasAce', data '{"armmode":0}' from endpoint 1 with groupID 0
zigbee2mqtt:info  2020-06-11 20:10:34: MQTT publish: topic 'zigbee2mqtt/0x000d6f000d8a5f63', payload '{"linkquality":5,"temperature":25.5,"keycode":"\u0000\u0000\u0000\u0000\u0000\u0000","voltage":2900,"action":"disarm"}'
  zigbee-herdsman:controller:endpoint DefaultResponse 0x000d6f000d8a5f63/1 1281(0, {"timeout":10000,"disableResponse":false,"disableDefaultResponse":true,"direction":1,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null}) +3s
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +3ms
zigbee2mqtt:info  2020-06-11 20:10:34: MQTT publish: topic 'zigbee2mqtt/0x000d6f000d8a5f63', payload '{"linkquality":5,"temperature":25.5,"keycode":"\u0000\u0000\u0000\u0000\u0000\u0000","voltage":2900,"action":""}'
zigbee2mqtt:info  2020-06-11 20:10:34: MQTT publish: topic 'homeassistant/device_automation/0x000d6f000d8a5f63/action_disarm/config', payload '{"automation_type":"trigger","type":"action","subtype":"disarm","payload":"disarm","topic":"zigbee2mqtt/0x000d6f000d8a5f63/action","device":{"identifiers":["zigbee2mqtt_0x000d6f000d8a5f63"],"name":"0x000d6f000d8a5f63","sw_version":"Zigbee2mqtt 1.14.0","model":"CentraLite 3400-D Keypad (3400-D)","manufacturer":"CentraLite"}}'
zigbee2mqtt:info  2020-06-11 20:10:34: MQTT publish: topic 'zigbee2mqtt/0x000d6f000d8a5f63/action', payload 'disarm'
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,32,68,129,0,0,1,5,255,61,1,1,0,15,0,173,114,79,0,0,12,1,100,0,0,6,49,50,51,52,53,54,0,255,61,29,11] +3s
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,32,68,129,0,0,1,5,255,61,1,1,0,15,0,173,114,79,0,0,12,1,100,0,0,6,49,50,51,52,53,54,0,255,61,29,11] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 32 - 2 - 4 - 129 - [0,0,1,5,255,61,1,1,0,15,0,173,114,79,0,0,12,1,100,0,0,6,49,50,51,52,53,54,0,255,61,29] - 11 +0ms
  zigbee-herdsman:adapter:zStack:znp:AREQ <-- AF - incomingMsg - {"groupid":0,"clusterid":1281,"srcaddr":15871,"srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"linkquality":15,"securityuse":0,"timestamp":5206701,"transseqnumber":0,"len":12,"data":{"type":"Buffer","data":[1,100,0,0,6,49,50,51,52,53,54,0]}} +3s
  zigbee-herdsman:controller:log Received 'zcl' data '{"frame":{"Header":{"frameControl":{"frameType":1,"manufacturerSpecific":false,"direction":0,"disableDefaultResponse":false,"reservedBits":0},"transactionSequenceNumber":100,"manufacturerCode":null,"commandIdentifier":0},"Payload":{"armmode":0}},"address":15871,"endpoint":1,"linkquality":15,"groupID":0}' +3s
zigbee2mqtt:debug 2020-06-11 20:10:37: Received Zigbee message from '0x000d6f000d8a5f63', type 'commandArm', cluster 'ssIasAce', data '{"armmode":0}' from endpoint 1 with groupID 0
zigbee2mqtt:info  2020-06-11 20:10:37: MQTT publish: topic 'zigbee2mqtt/0x000d6f000d8a5f63', payload '{"linkquality":15,"temperature":25.5,"keycode":"\u0000\u0000\u0000\u0000\u0000\u0000","voltage":2900,"action":"disarm"}'
  zigbee-herdsman:controller:endpoint DefaultResponse 0x000d6f000d8a5f63/1 1281(0, {"timeout":10000,"disableResponse":false,"disableDefaultResponse":true,"direction":1,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null}) +3s
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +2ms
zigbee2mqtt:info  2020-06-11 20:10:37: MQTT publish: topic 'zigbee2mqtt/0x000d6f000d8a5f63', payload '{"linkquality":15,"temperature":25.5,"keycode":"\u0000\u0000\u0000\u0000\u0000\u0000","voltage":2900,"action":""}'
zigbee2mqtt:info  2020-06-11 20:10:37: MQTT publish: topic 'zigbee2mqtt/0x000d6f000d8a5f63/action', payload 'disarm'
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,3,68,128,240,1,3,53] +510ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,3,68,128,240,1,3,53] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 3 - 2 - 4 - 128 - [240,1,3] - 53 +0ms
  zigbee-herdsman:adapter:zStack:znp:AREQ <-- AF - dataConfirm - {"status":240,"endpoint":1,"transid":3} +511ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +1ms
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,13,69,193,255,61,255,61,99,95,138,13,0,111,13,0,128,208] +470ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,13,69,193,255,61,255,61,99,95,138,13,0,111,13,0,128,208] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 13 - 2 - 5 - 193 - [255,61,255,61,99,95,138,13,0,111,13,0,128] - 208 +0ms
  zigbee-herdsman:adapter:zStack:znp:AREQ <-- ZDO - endDeviceAnnceInd - {"srcaddr":15871,"nwkaddr":15871,"ieeeaddr":"0x000d6f000d8a5f63","capabilities":128} +471ms
  zigbee-herdsman:controller:log Device announce '0x000d6f000d8a5f63' +982ms
zigbee2mqtt:debug 2020-06-11 20:10:38: Device '0x000d6f000d8a5f63' announced itself
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +1ms
zigbee2mqtt:info  2020-06-11 20:10:38: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"type":"device_announced","message":"announce","meta":{"friendly_name":"0x000d6f000d8a5f63"}}'
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,12,69,202,255,61,99,95,138,13,0,111,13,0,0,0,152] +132ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,12,69,202,255,61,99,95,138,13,0,111,13,0,0,0,152] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 12 - 2 - 5 - 202 - [255,61,99,95,138,13,0,111,13,0,0,0] - 152 +0ms
  zigbee-herdsman:adapter:zStack:znp:AREQ <-- ZDO - tcDeviceInd - {"nwkaddr":15871,"extaddr":"0x000d6f000d8a5f63","parentaddr":0} +133ms
  zigbee-herdsman:controller:log Device '0x000d6f000d8a5f63' joined +134ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +1ms
  zigbee-herdsman:controller:log Device '0x000d6f000d8a5f63' accepted by handler +0ms
  zigbee-herdsman:controller:log Not interviewing '0x000d6f000d8a5f63', completed 'true', in progress 'false' +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,13,69,193,255,61,255,61,99,95,138,13,0,111,13,0,128,208] +827ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,13,69,193,255,61,255,61,99,95,138,13,0,111,13,0,128,208] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 13 - 2 - 5 - 193 - [255,61,255,61,99,95,138,13,0,111,13,0,128] - 208 +0ms
  zigbee-herdsman:adapter:zStack:znp:AREQ <-- ZDO - endDeviceAnnceInd - {"srcaddr":15871,"nwkaddr":15871,"ieeeaddr":"0x000d6f000d8a5f63","capabilities":128} +828ms
  zigbee-herdsman:controller:log Device announce '0x000d6f000d8a5f63' +827ms
zigbee2mqtt:debug 2020-06-11 20:10:39: Device '0x000d6f000d8a5f63' announced itself
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +1ms
zigbee2mqtt:info  2020-06-11 20:10:39: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"type":"device_announced","message":"announce","meta":{"friendly_name":"0x000d6f000d8a5f63"}}'
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,32,68,129,0,0,1,5,255,61,1,1,0,15,0,182,140,79,0,0,12,1,100,0,0,6,49,50,51,52,53,54,0,255,61,29,238] +189ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,32,68,129,0,0,1,5,255,61,1,1,0,15,0,182,140,79,0,0,12,1,100,0,0,6,49,50,51,52,53,54,0,255,61,29,238] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 32 - 2 - 4 - 129 - [0,0,1,5,255,61,1,1,0,15,0,182,140,79,0,0,12,1,100,0,0,6,49,50,51,52,53,54,0,255,61,29] - 238 +0ms
  zigbee-herdsman:adapter:zStack:znp:AREQ <-- AF - incomingMsg - {"groupid":0,"clusterid":1281,"srcaddr":15871,"srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"linkquality":15,"securityuse":0,"timestamp":5213366,"transseqnumber":0,"len":12,"data":{"type":"Buffer","data":[1,100,0,0,6,49,50,51,52,53,54,0]}} +190ms
  zigbee-herdsman:controller:log Received 'zcl' data '{"frame":{"Header":{"frameControl":{"frameType":1,"manufacturerSpecific":false,"direction":0,"disableDefaultResponse":false,"reservedBits":0},"transactionSequenceNumber":100,"manufacturerCode":null,"commandIdentifier":0},"Payload":{"armmode":0}},"address":15871,"endpoint":1,"linkquality":15,"groupID":0}' +190ms
zigbee2mqtt:debug 2020-06-11 20:10:39: Received Zigbee message from '0x000d6f000d8a5f63', type 'commandArm', cluster 'ssIasAce', data '{"armmode":0}' from endpoint 1 with groupID 0
zigbee2mqtt:info  2020-06-11 20:10:39: MQTT publish: topic 'zigbee2mqtt/0x000d6f000d8a5f63', payload '{"linkquality":15,"temperature":25.5,"keycode":"\u0000\u0000\u0000\u0000\u0000\u0000","voltage":2900,"action":"disarm"}'
  zigbee-herdsman:controller:endpoint DefaultResponse 0x000d6f000d8a5f63/1 1281(0, {"timeout":10000,"disableResponse":false,"disableDefaultResponse":true,"direction":1,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null}) +2s
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +1ms
zigbee2mqtt:info  2020-06-11 20:10:39: MQTT publish: topic 'zigbee2mqtt/0x000d6f000d8a5f63', payload '{"linkquality":15,"temperature":25.5,"keycode":"\u0000\u0000\u0000\u0000\u0000\u0000","voltage":2900,"action":""}'
zigbee2mqtt:info  2020-06-11 20:10:39: MQTT publish: topic 'zigbee2mqtt/0x000d6f000d8a5f63/action', payload 'disarm'
  zigbee-herdsman:adapter:zStack:znp:SREQ --> AF - dataRequest - {"dstaddr":15871,"destendpoint":1,"srcendpoint":1,"clusterid":1281,"transid":4,"options":0,"radius":30,"len":5,"data":{"type":"Buffer","data":[24,99,11,7,0]}} +6s
  zigbee-herdsman:adapter:zStack:unpi:writer --> frame [254,15,36,1,255,61,1,1,1,5,4,0,30,5,24,99,11,7,0,132] +6s
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,1,100,1,0,100] +386ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,1,100,1,0,100] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 1 - 3 - 4 - 1 - [0] - 100 +0ms
  zigbee-herdsman:adapter:zStack:znp:SRSP <-- AF - dataRequest - {"status":0} +6s
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,3,68,128,0,1,4,194] +2s
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,3,68,128,0,1,4,194] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 3 - 2 - 4 - 128 - [0,1,4] - 194 +0ms
  zigbee-herdsman:adapter:zStack:znp:AREQ <-- AF - dataConfirm - {"status":0,"endpoint":1,"transid":4} +3s
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +0ms
  zigbee-herdsman:adapter:zStack:znp:SREQ --> AF - dataRequest - {"dstaddr":15871,"destendpoint":1,"srcendpoint":1,"clusterid":1281,"transid":5,"options":0,"radius":30,"len":5,"data":{"type":"Buffer","data":[24,100,11,0,0]}} +2s
  zigbee-herdsman:adapter:zStack:unpi:writer --> frame [254,15,36,1,255,61,1,1,1,5,5,0,30,5,24,100,11,0,0,133] +2s
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,23,68,129,0,0,1,5,255,61,1,1,0,34,0,27,175,79,0,0,3,1,101,7,255,61,29,114] +19ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,23,68,129,0,0,1,5,255,61,1,1,0,34,0,27,175,79,0,0,3,1,101,7,255,61,29,114] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 23 - 2 - 4 - 129 - [0,0,1,5,255,61,1,1,0,34,0,27,175,79,0,0,3,1,101,7,255,61,29] - 114 +0ms
  zigbee-herdsman:adapter:zStack:znp:AREQ <-- AF - incomingMsg - {"groupid":0,"clusterid":1281,"srcaddr":15871,"srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"linkquality":34,"securityuse":0,"timestamp":5222171,"transseqnumber":0,"len":3,"data":{"type":"Buffer","data":[1,101,7]}} +19ms
  zigbee-herdsman:controller:log Received 'zcl' data '{"frame":{"Header":{"frameControl":{"frameType":1,"manufacturerSpecific":false,"direction":0,"disableDefaultResponse":false,"reservedBits":0},"transactionSequenceNumber":101,"manufacturerCode":null,"commandIdentifier":7},"Payload":{}},"address":15871,"endpoint":1,"linkquality":34,"groupID":0}' +3s
  zigbee-herdsman:controller:log Skipping command 'getPanelStatus' because it is missing from the lookup +0ms
  zigbee-herdsman:controller:endpoint DefaultResponse 0x000d6f000d8a5f63/1 1281(7, {"timeout":10000,"disableResponse":false,"disableDefaultResponse":true,"direction":1,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null}) +3s
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,1,100,1,0,100] +1ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,1,100,1,0,100] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 1 - 3 - 4 - 1 - [0] - 100 +0ms
  zigbee-herdsman:adapter:zStack:znp:SRSP <-- AF - dataRequest - {"status":0} +2s
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,3,68,128,0,1,5,195] +1s
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,3,68,128,0,1,5,195] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 3 - 2 - 4 - 128 - [0,1,5] - 195 +0ms
  zigbee-herdsman:adapter:zStack:znp:AREQ <-- AF - dataConfirm - {"status":0,"endpoint":1,"transid":5} +1s
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +0ms
  zigbee-herdsman:adapter:zStack:znp:SREQ --> AF - dataRequest - {"dstaddr":15871,"destendpoint":1,"srcendpoint":1,"clusterid":1281,"transid":6,"options":0,"radius":30,"len":5,"data":{"type":"Buffer","data":[24,100,11,0,0]}} +2s
  zigbee-herdsman:adapter:zStack:unpi:writer --> frame [254,15,36,1,255,61,1,1,1,5,6,0,30,5,24,100,11,0,0,134] +2s
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,1,100,1,0,100] +10ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,1,100,1,0,100] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 1 - 3 - 4 - 1 - [0] - 100 +0ms
  zigbee-herdsman:adapter:zStack:znp:SRSP <-- AF - dataRequest - {"status":0} +1s
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,3,68,128,0,1,6,192] +494ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,3,68,128,0,1,6,192] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 3 - 2 - 4 - 128 - [0,1,6] - 192 +0ms
  zigbee-herdsman:adapter:zStack:znp:AREQ <-- AF - dataConfirm - {"status":0,"endpoint":1,"transid":6} +504ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +0ms
  zigbee-herdsman:adapter:zStack:znp:SREQ --> AF - dataRequest - {"dstaddr":15871,"destendpoint":1,"srcendpoint":1,"clusterid":1281,"transid":7,"options":0,"radius":30,"len":5,"data":{"type":"Buffer","data":[24,100,11,0,0]}} +504ms
  zigbee-herdsman:adapter:zStack:unpi:writer --> frame [254,15,36,1,255,61,1,1,1,5,7,0,30,5,24,100,11,0,0,135] +504ms
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,1,100,1,0,100] +10ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,1,100,1,0,100] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 1 - 3 - 4 - 1 - [0] - 100 +0ms
  zigbee-herdsman:adapter:zStack:znp:SRSP <-- AF - dataRequest - {"status":0} +504ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +1ms
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,3,68,128,0,1,7,193] +493ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,3,68,128,0,1,7,193] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 3 - 2 - 4 - 128 - [0,1,7] - 193 +0ms
  zigbee-herdsman:adapter:zStack:znp:AREQ <-- AF - dataConfirm - {"status":0,"endpoint":1,"transid":7} +504ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +0ms
  zigbee-herdsman:adapter:zStack:znp:SREQ --> AF - dataRequest - {"dstaddr":15871,"destendpoint":1,"srcendpoint":1,"clusterid":1281,"transid":8,"options":0,"radius":30,"len":5,"data":{"type":"Buffer","data":[24,101,11,7,0]}} +504ms
  zigbee-herdsman:adapter:zStack:unpi:writer --> frame [254,15,36,1,255,61,1,1,1,5,8,0,30,5,24,101,11,7,0,142] +504ms
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,1,100,1,0,100] +11ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,1,100,1,0,100] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 1 - 3 - 4 - 1 - [0] - 100 +0ms
  zigbee-herdsman:adapter:zStack:znp:SRSP <-- AF - dataRequest - {"status":0} +505ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +0ms
zigbee2mqtt:info  2020-06-11 20:10:45: Disconnecting from MQTT server
  zigbee-herdsman:controller:log Disable joining +3s
  zigbee-herdsman:adapter:zStack:znp:SREQ --> AF - dataRequestExt - {"dstaddrmode":2,"dstaddr":"0x000000000000fffd","destendpoint":242,"dstpanid":0,"srcendpoint":242,"clusterid":33,"transid":9,"options":0,"radius":30,"len":6,"data":{"type":"Buffer","data":[25,3,2,11,0,0]}} +592ms
  zigbee-herdsman:adapter:zStack:unpi:writer --> frame [254,26,36,2,2,253,255,0,0,0,0,0,0,242,0,0,242,33,0,9,0,30,6,0,25,3,2,11,0,0,31] +592ms
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,1,100,2,0,103] +594ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,1,100,2,0,103] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 1 - 3 - 4 - 2 - [0] - 103 +0ms
  zigbee-herdsman:adapter:zStack:znp:SRSP <-- AF - dataRequestExt - {"status":0} +594ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,3,68,128,0,242,9,60] +6ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,3,68,128,0,242,9,60] +0ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 3 - 2 - 4 - 128 - [0,242,9] - 60 +0ms
  zigbee-herdsman:adapter:zStack:znp:AREQ <-- AF - dataConfirm - {"status":0,"endpoint":242,"transid":9} +611ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +1ms
  zigbee-herdsman:adapter:zStack:znp:SREQ --> ZDO - mgmtPermitJoinReq - {"addrmode":15,"dstaddr":65532,"duration":0,"tcsignificance":0} +214ms
  zigbee-herdsman:adapter:zStack:unpi:writer --> frame [254,5,37,54,15,252,255,0,0,26] +213ms
  zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,1,101,54,0,82] +212ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,1,101,54,0,82] +1ms
  zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 1 - 3 - 5 - 54 - [0] - 82 +0ms
  zigbee-herdsman:adapter:zStack:znp:SRSP <-- ZDO - mgmtPermitJoinReq - {"status":0} +220ms
  zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [] +0ms
  zigbee-herdsman:adapter:zStack:znp:log Port closed +21s
zigbee2mqtt:info  2020-06-11 20:10:45: zigbee-herdsman stopped
```

Reply:
Could you also make a pr for the herdsman repo?

Reply:
> 
> 
> Could you also make a pr for the herdsman repo?

@sparky3387 would you please do that? See [Centralite 3400 keypad support](https://zigbee2mqtt.discourse.group/t/centralite-3400-keypad-support/697/12?u=marcus) on discourse

Reply:
Please see my changes in the merge and update this code accordingly

Reply:
Sorry, i can't update this code. I don't know how to do this. Maybe @sparky3387 can do this.
I've updated my `cluster.js` manually by adding and removing lines accordingly. I can't find `zcl.test.ts` in my zigbee2mqtt installation... i think thats not the right way how i'm doing this.
I might be a better support at sniffing the packets between device and original base.

Reply:
The only thing you have to do is to switch to the dev branch (https://www.zigbee2mqtt.io/how_tos/how-to-switch-to-dev-branch.html)